tests/test_bot.py

```diff
@@ -364,6 +368,7 @@ def test_get_chat_member(self, bot, channel_id):
         assert chat_member.status == 'administrator'
         assert chat_member.user.username == 'EchteEldin'
 
+    @pytest.mark.skipif(os.getenv('APPVEYOR'), reason='Skip game tests on Appveyor')
```

Comment:
Maybe we should write why we're skipping and not just that we are in the reason?
(I know we haven't really done this before but it makes more sense to do it like that I think)
(applies elsewhere)